core/src/main/scala/kafka/controller/KafkaController.scala

```diff
@@ -1472,6 +1525,49 @@ class KafkaController(val config: KafkaConfig, zkUtils: ZkUtils, val brokerState
     }
   }
 
+  case class LeaderAndIsrResponseReceived(LeaderAndIsrResponseObj: AbstractResponse, brokerId: Int) extends ControllerEvent {
```

Comment:
I think it may be better to rename `TopicDeletionStopReplicaResult` to `TopicDeletionStopReplicaResponseReceived`. But I don't have a strong opinion on this. I can also rename `LeaderAndIsrResponseReceived` to `LeaderAndIsrResponseResult`. Do you have a strong preference between the two?